src/Polly.Shared/PolicyResult.cs

```diff
@@ -35,12 +35,25 @@ internal PolicyResult(OutcomeType outcome, Exception finalException, ExceptionTy
         /// </summary>
         public Context Context { get; }
 
-        internal static PolicyResult Successful(Context context)
+        /// <summary>
+        /// Builds a successful <see cref="PolicyResult" />
+        /// </summary>
+        /// <param name="context">The policy context</param>
+        /// <returns></returns>
+        public static PolicyResult Successful(Context context)
         {
             return new PolicyResult(OutcomeType.Successful, null, null, context);
         }
 
-        internal static PolicyResult Failure(Exception exception, ExceptionType exceptionType, Context context)
+
+        /// <summary>
+        /// Builds a failed <see cref="PolicyResult" />
+        /// </summary>
+        /// <param name="exception">The exception</param>
+        /// <param name="exceptionType">The exception type</param>
+        /// <param name="context">The policy context</param>
+        /// <returns></returns>
```

Comment:
((suggest add, for 'returns'))

`A <see cref="PolicyResult" /> representing a failed execution through the policy.`